tests/models/timm/test_timm_image_classification.py

```diff
@@ -34,32 +34,45 @@ def _load_inputs(self):
         return input_batch
 
 
-@pytest.mark.parametrize(
-    "mode",
-    ["train", "eval"],
-)
+model_list = [
+    "tf_efficientnet_lite0.in1k",
+    "tf_efficientnet_lite1.in1k",
+    "tf_efficientnet_lite2.in1k",
+    "tf_efficientnet_lite3.in1k",
+    "tf_efficientnet_lite4.in1k",
+    "ghostnet_100.in1k",
+    "ghostnetv2_100.in1k",
+    "inception_v4.tf_in1k",
+    "mixer_b16_224.goog_in21k",
+    "mobilenetv1_100.ra4_e3600_r224_in1k",
+    "ese_vovnet19b_dw.ra_in1k",
+    "xception71.tf_in1k",
+    "dla34.in1k",
+    "hrnet_w18.ms_aug_in1k",
+]
+mode_list = ["train", "eval"]
+model_and_mode_list = []
```

Comment:
It is fairly hard to understand what’s marked with fail from this. 
sometimes it is better to write plain dumb code - let’s do this here so it’s very clear. Wdyt?